src/components/molecules/Common/Header/index.tsx

```diff
@@ -16,6 +16,8 @@ export interface Props {
   currentTeam?: Team;
   currentProject?: Project;
   sceneId?: string;
+  loading?: boolean;
+  hasMoreItem?: boolean;
```

Comment:
These two and handleScrolling below aren't being used.

Reply:
I moved those to settingPage